docs/Deploying-Pega-on-openshift.md

```diff
@@ -275,6 +275,16 @@ Configure the following parameters so the pega.yaml Helm chart matches your depl
 
 3. Save the file.
 
+#### (Optional) Add Support for providing DB credentials using External Secrets Operator
+
+Create two files following the Kubernetes documentation for External Secrets Operator [External Secrets Operator](https://external-secrets.io/v0.5.1/) :
```

Comment:
I think you addressed this in the next push, but it make sure everything states v0.5.3>>>https://external-secrets.io/v0.5.3/